<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeatNode</class>
 <widget class="QWidget" name="NeatNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>33</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="NodeCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="NodeLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="NodeTypeComboBox">
       <property name="currentText">
        <string>INPUT</string>
       </property>
       <item>
        <property name="text">
         <string>INPUT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HIDDEN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OUTPUT</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="NodeLayerComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Function</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="NodeFunctionComboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
